addons/modules/functions/fnc_gui_ambientFlyby.sqf

```diff
@@ -153,11 +158,11 @@ private _fnc_onConfirm = {
     private _ctrlAmount = _display displayCtrl IDC_AMBIENTFLYBY_AMOUNT;
     private _amount = lbCurSel _ctrlAmount;
 
-    GVAR(lastAmbientFlyby) = [_side, _faction, _aircraft, _direction, _height, _distance, _speed, _amount];
+    GVAR(lastAmbientFlyby) = [_side, _faction, _aircraft, _direction, _height, _useASL, _distance, _speed, _amount];
 
     [
         QGVAR(moduleAmbientFlyby),
-        [_aircraftType, ASLtoAGL getPosASL _logic, _height, _distance, _direction, _speed, _amount + 1]
+        [_aircraftType, getPosASL _logic, _height, _useASL == 1, _distance, _direction, _speed, _amount + 1]
```

Comment:
Trivial but, I think it would be better to keep the ordering so it follows the UI.